cratedb_toolkit/model.py

```diff
@@ -85,12 +85,15 @@ class TableAddress:
 
     @property
     def fullname(self):
-        if self.schema is None and self.table is None:
+        if self.table is None:
             raise ValueError("Uninitialized table address can not be serialized")
         if self.schema and self.table:
-            return f'"{self.schema}"."{self.table}"'
+            schema = self.schema.strip('"')
+            table = self.table.strip('"')
+            return f'"{schema}"."{table}"'
         else:
-            return f'"{self.table}"'
+            table = self.table.strip('"')
+            return f'"{table}"'
```

Comment:
This is totally ugly, but `DatabaseAdapter.quote_relation_name()` needs an SQLAlchemy engine, so it is not applicable here without further changes.

Reply:
Improved with da5fb377680.

Reply:
Diverted to a different patch.

- GH-202

Reply:
The section in the code has been improved using GH-202.